doc-api-examples/src/main/resources/python/model-arimax/new.rst

```diff
@@ -0,0 +1,130 @@
+
+Consider the following model trained and tested on the sample data set in *frame* 'frame'.
+The frame has five columns where "y" is the time series value and "vistors", "wkends",
+"incidentRate", and "seasonality" are exogenous inputs.
+
+<hide>
+>>> import trustedanalytics as ta
+>>> ta.connect()
+-etc-
+>>> schema = [("y", ta.float64),("visitors", ta.float64),("wkends", ta.float64),("seasonality", ta.float64),
```

Comment:
This example is still using Kyle's data - can you switch to use the air quality data (and include their citation note/link)